src/status_im/contexts/onboarding/sync_or_recover_profile/view.cljs

```diff
@@ -0,0 +1,117 @@
+       :max-height   (- (:height window)
+                        (* 2 56) ;; two other list items
+                        (* 2 16) ;; spacing between items
+                        220)     ;; extra spacing (top bar)
+       :on-press     #(debounce/throttle-and-dispatch
+                       [:onboarding/navigate-to-sign-in-by-syncing]
+                       1000)}]
+     [rn/view {:style style/subtitle-container}
+      [quo/text
+       {:style  style/subtitle
+        :size   :paragraph-2
+        :weight :medium}
+       (i18n/label :t/dont-have-statatus-on-another-device)]]
+     [rn/view
+      [quo/small-option-card
+       {:variant  :icon
+        :title    (i18n/label :t/use-recovery-phrase)
+        :subtitle (i18n/label :t/use-recovery-phrase-subtitle)
+        :image    (resources/get-image :ethereum-address)
+        :on-press #(rf/dispatch [:navigate-to-within-stack
+                                 [:screen/onboarding.enter-seed-phrase
+                                  :screen/onboarding.sync-or-recover-profile]])}]
+      [rn/view {:style style/space-between-suboptions}]
+      (when config/show-not-implemented-features?
+        [quo/small-option-card
+         {:variant  :icon
+          :title    (i18n/label :t/use-keycard)
+          :subtitle (i18n/label :t/use-keycard-subtitle)
+          :image    (resources/get-image :use-keycard)
+          :on-press #(js/alert "TODO: to be implemented")}])]]))
+
+(defn getting-started-doc
+  []
+  [quo/documentation-drawers
+   {:title  (i18n/label :t/getting-started-with-status)
+    :shell? true}
+   [:<>
+    [quo/text
+     {:size  :paragraph-2
+      :style style/doc-main-content}
+     (i18n/label :t/getting-started-description)]
+    [quo/text
+     {:size   :paragraph-1
+      :style  style/doc-subtitle
+      :weight :semi-bold}
+     (i18n/label :t/generate-keys)]
+    [quo/text
+     {:size  :paragraph-2
+      :style style/doc-content}
+     (i18n/label :t/getting-started-generate-keys-description)]
+    [quo/text
+     {:size   :paragraph-1
+      :style  style/doc-subtitle
+      :weight :semi-bold}
+     (i18n/label :t/getting-started-generate-keys-from-recovery-phrase)]
+    [quo/text
+     {:size  :paragraph-2
+      :style style/doc-content}
+     (i18n/label :t/getting-started-generate-keys-from-recovery-phrase-description)]
+    [quo/text
+     {:size   :paragraph-1
+      :style  style/doc-subtitle
+      :weight :semi-bold}
+     (i18n/label :t/getting-started-generate-keys-on-keycard)]
+    [quo/text
+     {:size  :paragraph-2
+      :style style/doc-content}
+     (i18n/label :t/getting-started-generate-keys-on-keycard-description)]]])
+
+(defn screen
+  []
+  (let [{:keys [top]} (safe-area/get-insets)]
+    [rn/view {:style style/content-container}
+     [quo/page-nav
+      {:margin-top top
+       :type       :no-title
+       :background :blur
+       :icon-name  :i/arrow-left
+       :on-press   #(do
```

Comment:
It's preferable in our views to cache these anonymous functions, either with `rn/use-callback` or by separate vars when possible. Otherwise Reagent props diff will always be false on every pass of `screen` and it will always re-process `page-nav`. It's not a big deal here because this screen is hardly re-rendering, but what if someone down the line makes this view more reactive? That person will likely not update the code to cache functions, so it's better if we try do it now.